frontend/src/pages/user/LessonResult.tsx

```diff
@@ -0,0 +1,106 @@
+import axios from 'axios';
+import { useMemo, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { Stack, Table } from 'react-bootstrap'
+import Result from '../../interfaces/result';
+import { Category } from '../../interfaces/category';
+import { Lesson } from '../../interfaces/lesson';
+import { Answer } from '../../interfaces/answer';
+import { getLesson } from '../../apiClient/lessonService';
+import { getCategory } from '../../apiClient/categoryService';
+import { listAnswersByLesson } from '../../apiClient/answerService';
+import { getQuestion } from '../../apiClient/questionService';
+
+function LessonResult() {
+  let [searchParams] = useSearchParams();
+  const lessonId = Number(searchParams.get("lesson"));
+
+  const [lesson, setLesson] = useState<Lesson>({
+    id: 0,
+    user: 0,
+    category: 0,
```

Comment:
rename to a more clearer description
```suggestion
    userId: 0,
    categoryId: 0,
```